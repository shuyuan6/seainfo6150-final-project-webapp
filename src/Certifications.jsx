import React from 'react'
import styles from "./Certifications.module.css"

const Certifications = () => {
    return (
        <div className = {styles.container}>
            <h1>Certifications</h1>
            <ul>
                <li>
                Certified by Cerner Multum Information Services: medications information database.
                </li>
                <br/>
                <li>
                Certified by Micromedex: Advanced Consumer Information database, Care Guide and Image Database.

                </li>
                <br/>

                <li>
                Certified by Physicians' Desk Reference: medications information database.

                </li>
                <br/>

                <li>
                Certified by Wolters Kluwer Health: facts and Comparisons Integrated MedFacts consumer medications information.

                </li>

            </ul>
        </div>
    )
}

export default Certifications