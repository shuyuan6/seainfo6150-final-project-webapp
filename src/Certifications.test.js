import React from "react";
import { render } from "@testing-library/react";
import Certifications from "./Certifications.jsx";

it("renders the Certifications component correctly", () => {
  const { container } = render(<Certifications />);
  expect(container).toMatchSnapshot();
});
