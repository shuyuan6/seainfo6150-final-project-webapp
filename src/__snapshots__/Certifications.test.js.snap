// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Certifications component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Certifications
    </h1>
    <ul>
      <li>
        Certified by Cerner Multum Information Services: medications information database.
      </li>
      <br />
      <li>
        Certified by Micromedex: Advanced Consumer Information database, Care Guide and Image Database.
      </li>
      <br />
      <li>
        Certified by Physicians' Desk Reference: medications information database.
      </li>
      <br />
      <li>
        Certified by Wolters Kluwer Health: facts and Comparisons Integrated MedFacts consumer medications information.
      </li>
    </ul>
  </div>
</div>
`;
